site/source/user-manual/service-guides/lightning/index.rst

```diff
@@ -11,14 +11,33 @@ The following guide will get you up and running with a lightning node and walk y
 
   lightning-first-channel
 
-Now that you're up and running, it's time to connect some wallets to your node so that you can use lightning from anywhere!
+Now that you're up and running, it's time to connect a wallets to your node so that you can operate your node and use lightning to make and receive payments from anywhere!
```

Comment:
```suggestion
Now that you're up and running, it's time to connect an interface so that you can use your node to make and receive payments from anywhere!
```